docs/seq-msi/registers.md

```diff
@@ -33,28 +33,28 @@ The binary-data, in a register, can be transfered within itself from one flip-fl
 A shift register is a type of register that allows such data transfers.
 Shift register has 4 modes of operations.
 
-## Sub-modules
+<!-- ## Sub-modules -->
```

Comment:
I believe you can simply remove them instead of commenting ?

Reply:
Sure, at first I didn't remove it in case it was preferred to get those heading back after reviewing the PR.